projects/common/src/utilities/formatters/date/date-formatter.ts

```diff
@@ -72,11 +76,23 @@ export class DateFormatter {
       return '-';
     }
 
-    return formatDate(coercedValue, this.getFormatString(), 'en_US');
+    return formatDate(coercedValue, this.getFormatString(coercedValue), 'en_US');
   }
 
-  private getFormatString(): string {
+  private getFormatString(date: Date): string {
     switch (this.options.mode) {
+      case DateFormatMode.TimeWithTimeZoneOffset:
+        /*
+         * WARNING: This format is currently unsupported by at least some backend APIs e.g. Reporting
```

Comment:
Not relevant here

Reply:
Eeeeh, I think its useful here too, but its fine. Removed.